bin/verifyBytecodes.ts

```diff
@@ -0,0 +1,116 @@
+import { execSync } from "node:child_process";
```

Comment:
ended up putting this file in `bin/` because I didn't want to create a `scripts` folder next to the already existing foundry `script` folder, but open to suggestion on where to put it.

Reply:
I think it's fine here 👍